dev-scripts/test-inflight-staging-contention-e2e.py

```
@@ -0,0 +1,590 @@
+    ]
+    if cdc:
+        args.append("--enable-cdc")
+    log(
+        f"start_cluster: db={db} storage={storage} replicas={replicas} "
+        f"cdc={cdc} locker=redis inflight-staging=on",
+        G,
+    )
+    f = open(log_path, "w")
+    # start_new_session=True isolates the child in its own process group so
+    # stop_cluster can kill the whole tree via os.killpg.
+    p = subprocess.Popen(
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)